test/powershell/engine/Module/ModulePath.Tests.ps1

```diff
@@ -93,4 +93,19 @@ Describe "SxS Module Path Basic Tests" -tags "CI" {
         $paths -contains $fakePSHomeModuleDir | Should Be $true
         $paths -contains $customeModules | Should Be $true
     }
+
+    It "Default PowerShell profile appends Windows PowerShell PSModulePath only on Windows" {
+
+        $psmodulepath = & $powershell -c '$env:PSModulePath'
```

Comment:
Like in this test case, would other tests now have the warning message and fail? Can you verify by running all tests?